pkg/capacityscheduling/capacity_scheduling_test.go

```diff
@@ -112,6 +85,41 @@ func TestPreFilter(t *testing.T) {
 				framework.Unschedulable,
 			},
 		},
+		{
+			name: "the sum of used is bigger than the sum of min",
+			podInfos: []podInfo{
+				{podName: "ns2-p1", podNamespace: "ns2", memReq: 500},
+			},
+			elasticQuotas: map[string]*ElasticQuotaInfo{
+				"ns1": {
+					Namespace: "ns2",
+					Min: &framework.Resource{
+						Memory: 1000,
+					},
+					Max: &framework.Resource{
+						Memory: 2000,
+					},
+					Used: &framework.Resource{
+						Memory: 1800,
+					},
+				},
+				"ns2": {
+					Namespace: "ns2",
+					Min: &framework.Resource{
+						Memory: 1000,
+					},
+					Max: &framework.Resource{
+						Memory: 2000,
+					},
+					Used: &framework.Resource{
+						Memory: 300,
```

Comment:
Although it's a UT, making it `200` is more reasonable (currently summed Used is 300 + 1800 > summed Min)